### PR TITLE
feat(24.04): add 'copyright' slice to all SDFs

### PR DESCRIPTION
**Pull request by cjdcordeiro**
# Proposed changes

A new `copyright` slice is being added to all Slice Definitions Files, for packages that actually install a copyright file.
In addition, this copyright file now becomes a global dependency (`essential`) for all slices within the SDF, via the top-level `essential` field.

## Related issues/PRs

https://github.com/canonical/chisel/pull/127

## Checklist

* [x] I have read the [contributing guidelines](
https://github.com/canonical/chisel-releases/blob/main/CONTRIBUTING.md)
* [x] I have already submitted the [CLA form](
https://ubuntu.com/legal/contributors/agreement)

### CODE REVIEW

File: slices/aspnetcore-runtime-8.0.yaml
Comment (cjdcordeiro) [reply slices/aspnetcore-runtime-8.0.yaml]: changed in https://github.com/canonical/chisel-releases/pull/228/commits/1fdf7dd1ea9a933cd0a618b74e05b89f89ac21fc

File: slices/liblttng-ust1t64.yaml
Comment (rebornplusplus) [inline slices/liblttng-ust1t64.yaml:4]: nit: it would be nice to have a blank line in between `package` and `essential`.

**Review by rebornplusplus — APPROVED** [review]
Looks good to me. The packages who have copyright files within them, do have the copyright slices added in this PR.

The files who do not have the copyright files for this release are the following:
* libatomic1
* libgcc-s1
* libncursesw6
* libpython3.12-stdlib
* libstdc++6

I only left a nit comment about spacing inside the slice definition files. But please feel free to ignore that -- it's a nitpick.

File: slices/liblttng-ust-ctl5t64.yaml
Comment (cjdcordeiro) [inline slices/liblttng-ust-ctl5t64.yaml:2]:
```suggestion

essential:
```

File: slices/liblttng-ust1t64.yaml
Comment (cjdcordeiro) [inline slices/liblttng-ust1t64.yaml:2]:
```suggestion

essential:
```

File: slices/liblzma5.yaml
Comment (cjdcordeiro) [inline slices/liblzma5.yaml:2]:
```suggestion

essential:
```

File: slices/libnuma1.yaml
Comment (cjdcordeiro) [inline slices/libnuma1.yaml:2]:
```suggestion

essential:
```

File: slices/libunwind8.yaml
Comment (cjdcordeiro) [inline slices/libunwind8.yaml:2]:
```suggestion

essential:
```

File: slices/liblttng-ust1t64.yaml
Comment (cjdcordeiro) [reply slices/liblttng-ust1t64.yaml:5]: done

**Review by rebornplusplus — APPROVED** [review]
Nice!

Do you wanna merge #232 first and give that a try in this PR?

**Comment by cjdcordeiro** [comment]
> Nice!
> 
> Do you wanna merge #232 first and give that a try in this PR?

Yes. Gonna wait for a final :+1: from @letFunny after his last comments, but since it has both approvals, I'll merge it anyway later in the morning

**Comment by letFunny** [comment]
I was waiting for #232 to approve this one :D

**Review by letFunny — APPROVED** [review]
Thanks!

**Pull request merged**
